components/movementsensor/dualgps/dualgps.go

```diff
@@ -0,0 +1,231 @@
+package dualgps
+
+import (
+	"context"
+	"math"
+	"sync"
+
+	"github.com/golang/geo/r3"
+	geo "github.com/kellydunn/golang-geo"
+	"go.viam.com/rdk/components/movementsensor"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/spatialmath"
+	"go.viam.com/rdk/utils"
+)
+
+// the default offset between the two gps devices describes a setup where
+// one gps is mounted on the right side of the base and one gps is mounted on the left side of the base
+//   ___________
+//  |	base	|
+//  |			|
+// GPS2			GPS1
+//  |			|
```

Comment:
oh eww, it's not showing up like this after listing on my machine

Reply:
Ah, you're using tabs, not spaces. For whitespace that's not at the beginning of the line, use spaces, and then it'll look the same everywhere.

(side note: this is why languages like Python advocate for spaces instead of tabs everywhere. In these modern times, there is approximately no reason tabs should exist in human-readable text, and I don't understand why the creators of Go thought otherwise. The strongest argument I've heard is that their whitespace is 4 times more storage-efficient, which is a bad argument given how plentiful storage is these days.)